keps/sig-node/4369-allow-special-characters-environment-variable/kep.yaml

```diff
@@ -18,12 +18,12 @@ see-also: []
 replaces: []
 
```

Comment:
you can add me as approver from SIG Node

Reply:
Added.